docs/operator-manual/upgrading/1.8-1.9.md

```diff
@@ -0,0 +1,7 @@
+# v1.8 to 1.9
```

Comment:
Intentionally not mentioning upgrade instructions in TOC to avoid confusion.

Reply:
I like the idea to use this document for tracking changes that need attention during upgrade together with the PR that introduces such a change. We should keep this habit :+1: